fec.config.js

```diff
@@ -0,0 +1,28 @@
+const { resolve } = require('path');
+
+module.exports = {
+    appUrl: '/insights/patch',
+    debug: true,
+    useProxy: process.env.PROXY === 'true',
+    proxyVerbose: true,
+    interceptChromeConfig: false,
+    plugins: [],
+    ...(process.env.HOT ? { hotReload: process.env.HOT === 'true' } : { hotReload: true }),
+    ...(process.env.port ? { port: parseInt(process.env.port) } : {}),
+    moduleFederation: {
+        shared: [
+            {
+                'react-router-dom': {
+                    singleton: true,
+                    import: false,
+                    version: '^6.8.1',
+                    requiredVersion: '^6.8.1'
```

Comment:
I am wondering what the best practice is with these version props and what would be the best to avoid running into issues. 

I'm thinking maybe they should be a bit more open. Similar to the sem ver definitions: https://github.com/RedHatInsights/frontend-components/blob/master/packages/components/package.json#L45-L46

For the `requiredVersion` we could use `>=6.0.0 <7.0.0` and I think we can drop the `version`, as webpack will default to the version in `package.json`

Reply:
is not `requiredVersion: '^6.8.1' `equal to `requiredVersion: ' >=6.0.0 <7.0.0'`?

Reply:
I'm not sure we need it to be exactly higher than `6.8.1`, or?

In general unless we have a very specific reason to stick to a very specific version, we should keep them "unspecific". 6.8.1 is not even the version that is in the lock file, it is `6.22.3`, or in the package.json, which is `^6.11.1`,

This leads to confusion, which is the correct version. Let's stick to `>=X.0.0` definitions. Then we can also just copy and paste these things, as they'll apply equally in each project.